src/NuGet.Clients/PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -387,5 +466,65 @@ public IEnumerable<NuGetProject> NuGetProjects
                 return _nugetProjects;
             }
         }
+
+        protected void AddBlockedVersions(NuGetVersion[] blockedVersions)
+        {
+            // add a separator
+            if (blockedVersions.Length > 0)
```

Comment:
this can't be null, since this isn't a public method I think the expectation that this is already valid within the class is fine. thoughts?